### ISSUE TITLE
Issues with MA0003 auto code fix

### ISSUE THREAD

**Issue by areevesat** [meziantou/Meziantou.Analyzer#1167]
### Version of the Meziantou.Analyzer NuGet package

3.0.90

### Rule Identifier

MA0003

### Target Framework

.NET 10.0

### C# Language version

14.0

### Description

The auto code fix for MA0003 appears to malfunction when multiple parameters in a single method call need to have names added.

### Reproduction Steps

Minimal code:

```C#
await this.InsertStatus(null, null, null, utcStatuses, null);
```

Generates the below result:

```C#
await this.InsertStatus(reviewStatuses: courseStatuses: paymentInfos: null, null, utcStatuses, dmvStatuses: null);
```

Those are correct parameter names, but they have been applied incorrectly.
